src/main/cpp/subsystems/Flywheel.cpp

```diff
@@ -105,6 +110,11 @@ units::radians_per_second_t Flywheel::GetReferenceForPose(
         TargetModel::kTargetPoseInGlobal.Translation())];
 }
 
+units::radians_per_second_t Flywheel::GetReferenceForRange(
+    const units::meter_t& range) const {
```

Comment:
Pass the range parameter by value instead of const ref since it's just a double.